lua/conform/formatters/dockerfmt.lua

```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/reteps/dockerfmt",
+    description = "Dockerfile formatter. a modern dockfmt.",
+  },
+  command = "dockerfmt",
+  args = { "$FILENAME" },
```

Comment:
It looks like to format from stdin you should pass no arguments https://github.com/reteps/dockerfmt/pull/23

Reply:
I've updated the code.